packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx

```diff
@@ -54,103 +54,129 @@ type TreeNodeRendererProps = Pick<ReturnType<typeof useTree>, "expandNode" | "is
  * @see https://itwinui.bentley.com/docs/tree
  * @public
  */
-export function TreeNodeRenderer({
-  node,
-  expandNode,
-  getIcon,
-  getLabel,
-  onFilterClick,
-  onNodeClick,
-  onNodeKeyDown,
-  isNodeSelected,
-  getHierarchyLevelDetails,
-  reloadTree,
-  actions,
-  ...treeItemProps
-}: TreeNodeRendererProps) {
-  const nodeRef = useRef<HTMLDivElement>(null);
+export const TreeNodeRenderer: React.ForwardRefExoticComponent<TreeNodeRendererProps & RefAttributes<HTMLDivElement>> = forwardRef(
```

Comment:
Can we use `HTMLElement` instead of `HTMLDivElement`?

Reply:
changed